### ISSUE TITLE
Shell terminal cropped by on-screen keyboard

### ISSUE THREAD

**Issue by Altair31** [cutiepi-io/cutiepi-shell#21]
The shell interface does not resize when on-screen keyboard is shown.
On terminal, we cannot check our input while we are typing if we got lot of output from a previous command (like apt update).

Installed version: CutiePi image - 2022-01-23

**Comment by imaclmaca** [comment]
Yes - this bothers me greatly too; I do a lot at the terminal, and the OSK covering the terminal is really a bad feature.
